platforms/common/include/px4_platform_common/px4_work_queue/WorkQueueManager.hpp

```diff
@@ -64,6 +64,8 @@ static constexpr wq_config_t I2C4{"wq:I2C4", 1250, -10};
 
 static constexpr wq_config_t att_pos_ctrl{"wq:att_pos_ctrl", 6600, -11}; // PX4 att/pos controllers, highest priority after sensors
 
+static constexpr wq_config_t uavcan{"uavcan", 2400, -13};
```

Comment:
Any particular reason this settled here just below hp_default?

Reply:
Nothing in particular. Since it's using a lot of CPU I did not want it to interfere with other critical tasks. But since it's an output driver, it could also be higher.